src/rcl_bindings.cpp

```diff
@@ -696,6 +696,10 @@ NAN_METHOD(InitString) {
     info.GetReturnValue().Set(Nan::Undefined());
 }
 
+inline char* get_addr(v8::Local<v8::Value> buf) {
```

Comment:
As `get_addr` doesn't return a attribute of a class, like set/get pair, shall we use `GetAddress` to make the convention consistent. Others are good to me.